src/infrasys/system.py

```diff
@@ -1094,6 +1095,12 @@ def merge_system(self, other: "System") -> None:
 
     # TODO: add delete methods that (1) don't raise if not found and (2) don't return anything?
 
+    def convert_storage(self, **kwargs) -> TimeSeriesStorageBase | None:
```

Comment:
This always returns `None` now, right? 

Also, would you mind adding an example, like when converting to in-memory? Users are probably not going to immediately know what to pass.